minecraft_servers/unicacity/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "server_name": "unicacity",
+  "nice_name": "UnicaCity",
+  "direct_ip": "server.unicacity.de",
+  "server_wildcards": [
+    "*.unicacity.de"

```

Comment:
Ahh, there is still one thing.
This has to be `%.unicacity.de` :)

Reply:
done